src/server/array_func.c

```diff
@@ -682,7 +682,7 @@ create_subjob(job *parent, char *newjid, int *rc)
 	svrattrl  *psatl;
 	job 	  *subj;
 	long	   eligibletime;
-	long	    time_msec;
+	long	    time_usec;
```

Comment:
I know I wrote time_msec but I think it was a mistake not making it an unsigned long. If you want, you can do that. Not that it matters because we will be all gone before it fills up :)